docs/notebooks/ask_tell_optimization.pct.py

```diff
@@ -168,4 +168,23 @@ def plot_ask_tell_regret(ask_tell_result):
 
 
 # %% [markdown]
-# A word of warning. This serialization technique is not guaranteed to work smoothly with every Tensorflow-based model, so apply to your own problems with caution.
+# In some more complicated scenarios we may also wish to serialise the acquisition function, rather than creating a new one from the models and data, as it may contain stochastic internal data. This is not an issue here (where we used the default `EfficientGlobalOptimization` rule and `ExpectedImprovement` function) but we can demonstrate it neverthless:
```

Comment:
perhaps mention an acq function where there is an internal state?

Reply:
Can you remind me what the archetypal use case for this was?

Reply:
any that is using trajectory samplers I think, bcs they have internal state acq func has stochastic internal state... not sure about any others, perhaps one of the Henry's acq functions has internal states?